static/scripts/rewards/render-transaction/render-transaction.ts

```diff
@@ -63,6 +63,7 @@ export async function renderTransaction(): Promise<Success> {
   } else {
     const requestedAmountElement = insertErc721PermitTableData(app.reward, table);
     table.setAttribute(`data-make-claim`, "ok");
+    table.setAttribute(`data-additional-data-size`, "large");
```

Comment:
Where is the logic to remove this? If this is to display on NFT rewards, what happens when you switch to a non NFT reward? Does it re-render the table without this attribute?

Reply:
This scenario never dawned on me but it does not re-render the table. I crafted a permit with both types in it NFT first and it shifted for ERC20.